assets/src/scss/new-identity/style.scss

```diff
@@ -52,6 +52,22 @@
   --block-spreadsheet-blue-header--background: #167f82;
   --block-spreadsheet-blue-odd-row--background: var(--beige-200);
   --block-spreadsheet-blue-even-row--background: var(--beige-100);
+  --table-header-grey--background: var(--grey-800);
```

Comment:
It just occurred to me that these values are the same as for the Spreadsheet block, maybe we could use the `--table` variables for both blocks? 🤔 That would simplify the code quite a bit

Reply:
Yea, that is true, I will refactor and make the necessary changes